server.lua

```diff
@@ -89,9 +126,32 @@ RegisterServerEvent('dream-postal:server:log', function(data)
         { ['Content-Type'] = 'application/json' })
 end)
 
+AddEventHandler("playerDropped", function()
+    if currentWorkers[source] then
+        currentWorkers[source] = nil
+    end
+
+    TriggerEvent("dream-postal:server:log", { source = source, type = "success", message = t('ended_shift')})
+end)
+
 
 -- utils
 
+function IsNearAnyDeliverPoint(source)
```

Comment:
Not a showstopper, I like lowercase convention to differentiate from natives doesn't need to be done in this pr i can do us a follow up commit

```suggestion
function isNearAnyDeliverPoint(source)
```